client/blocks/product-purchase-features/product-purchase-features-list/upload-plugins.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { selectedSite, translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				icon="plugins"
+				title={ translate( 'Upload a Plugin' ) }
+				description={ translate(
+					'You can now upload your own plugins, or ones you\'ve downloaded, directly through a drag and drop interface.'
+				) }
+				buttonText={ translate( 'Upload a plugin now' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).